examples/index.html

```diff
@@ -232,7 +232,7 @@ <h2>Improve this tool</h2>
         var sequencer;
       })
     </script>
-
+    <script async src="./opencv_3.3.1.js" type="text/javascript"></script>
```

Comment:
Oh, i still want to grab this file via NPM instead of including it in full. Let me check through the links we found but if there is /any/ git repo where this is available, we can pull that in using NPM and link it in the `node_modules` folder. That'll keep this commit smaller!

Reply:
So it'll be

```suggestion
    <script async src="../node_modules/opencvjs/build/cv.js" type="text/javascript"></script>
```

Reply:
OK, and I added a commit linking to the latest commit (from 2017) in `package.json`. Pull this to your local and you should be able to install it in `node_modules` -- tell me if this works! Thanks!

Reply:
Oh, and if it works, we can delete the fully-included file at `examples/opencv_3.3.1.js`! 🎉